src/collectors/REFERENCE.md

```diff
@@ -64,7 +64,7 @@ After you make your changes, restart the Agent with the [appropriate method](/do
 
 Every Collector and Plugin have their own `update every` settings, which you can also change in their respective configuration files.
 
-To reduce the collection frequency of a [Plugin](/src/collectors/README.md#collector-architecture-and-terminology), open `netdata.conf` using [`edit-config`](/docs/netdata-agent/configuration/README.md#edit-a-configuration-file-using-edit-config) and find the appropriate section.
+To reduce the collection frequency of a plugin, open `netdata.conf` using [`edit-config`](/docs/netdata-agent/configuration/README.md#edit-a-configuration-file-using-edit-config) and find the appropriate section.
```

Comment:
@ilyam8 should we uppercase "Plugins" and "Collectors"?

I think we can add it to the dictionary too if we choose to

Reply:
I am not sure.